test/fiddle/test_cparser.rb

```diff
@@ -277,6 +277,10 @@ def test_struct_string
       assert_equal [[TYPE_INT,TYPE_VOIDP,TYPE_VOIDP], ['x', 'cb', 'name']], parse_struct_signature('int x; void (*cb)(); const char* name')
     end
 
+    def test_struct_bool
+      assert_equal [[TYPE_INT, TYPE_BOOL], ['x', 'toggle']], parse_struct_signature('int x; bool toggle')
```

Comment:
Could you ensure using `(...)` for newly added assertions to avoid warnings?
See also:  https://bugs.ruby-lang.org/issues/20922

```suggestion
      assert_equal([[TYPE_INT, TYPE_BOOL], ['x', 'toggle']],
                   parse_struct_signature('int x; bool toggle'))
```